src/config/pages.ts

```diff
@@ -24,25 +24,30 @@ export const PAGE_CATEGORIES: PageCategories = [
   {
     _id: 1,
     title: 'default',
+    ctitle: 'default',
```

Comment:
`ctitle` should be present in a translation file. The page structure should not contain translation content. Can you separate these into the translation files and import the correct language into `title`?